static/styles/files/files.scss

```diff
@@ -119,40 +126,47 @@
 
     .file-preview {
       width: 90%;
-      height: 65px;
-      margin-right: 0;
+      height: 35px;
+      float: left;
+      margin-right: 5px;
       background: no-repeat center center;
       background-size: cover;
     }
 
     .fa {
-      margin-top: 10px;
-      margin-right: 15px;
+      padding: 7px 7px 0 7px;
       vertical-align: text-top;
       color: #ccc;
       &:hover {
         color: $primaryColor;
       }
     }
     .card-block {
+      padding: 10px;
       a:hover {
         text-decoration: none;
       }
+
+      .card-title{
+        display: flex;
+        align-items: center;
+        margin-bottom: 0;
+      }
     }
     .title {
       white-space: nowrap;
-      width: calc(75% - 30px);
+      width: calc(90%);
```

Comment:
ist das hier so gewollt? klappt das in allen Auflösungen?

Reply:
`calc` könnte auf jeden Fall weggelassen werden